src/extension-test/unit/cursor-doc/token-cursor-test.ts

```diff
@@ -76,11 +77,26 @@ describe('Token Cursor', () => {
             expect(cursor.offsetStart).toBe(14);
         });
 
+        xit('Does not move past unbalanced top level form', () => {
+            //TODO: Figure out why this doesn't work
+            const d = docFromTextNotation('|(foo "bar"');
+            const cursor: LispTokenCursor = d.getTokenCursor(d.selectionLeft);
+            cursor.forwardSexp();
+            expect(cursor.offsetStart).toBe(d.selectionLeft);
+        });
+        xit('Does not move past unbalanced top level form', () => {
```

Comment:
@bpringe, this is the test that breaks coming tests. Something runs amok when there is no closing quote. But the way it runds amok!

Reply:
Weird, so, maybe something to do with the `"` after `bar`, whereas the one above doesn't have that? No clue what it could be though.